### PR TITLE
Use docker-compose for orchestrating benchmarking

### PR DESCRIPTION
**Pull request by oscarbatori**

### CODE REVIEW

File: benchmark-tools/dolt-builds/start_dolt_sql_server.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (coffeegoddd) [inline benchmark-tools/dolt-builds/start_dolt_sql_server.sh:1]: `chmod +x`

File: benchmark-tools/sysbench/benchmark.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (coffeegoddd) [inline benchmark-tools/sysbench/benchmark.sh:1]: `chmod +x`

File: benchmark-tools/dolt-builds/start_dolt_sql_server.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+echo "Creating data directory and configuring Dolt"
+mkdir /test
+cd /test || return
+dolt config --global --add user.name benchmark
+dolt config --global --add user.email benchmark@dolthub.com
+dolt init
+dolt sql-server --host=0.0.0.0
```
Comment (reltuk) [inline benchmark-tools/dolt-builds/start_dolt_sql_server.sh:9]: This seems like it probably wants `set -e`? Something like:

```sh
set -e
echo "Creating data directory and configuring Dolt"
[ -d /test ] || mkdir /test
cd /test
dolt config --global --add user.name benchmark
dolt config --global --add user.email benchmark@dolthub.com
dolt init
exec dolt sql-server --host=0.0.0.0
```

File: benchmark-tools/mysql/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD="no"
```
Comment (reltuk) [inline benchmark-tools/mysql/docker-compose.yml:6]: This is confusing. Should this be `- MYSQL_ALLOW_EMPTY_PASSWORD="yes"`, or something that makes it clear that we actually are allowing an empty password and we're going to use it to connect to the database?

File: benchmark-tools/run_benchmarks.sh [hunk 5]
```diff
@@ -2,9 +2,9 @@
 set -e
 set -o pipefail
 
-[ ! -z "$1" ] || (echo "Please supply a comma separated list of tests to be run"; exit 1)
+[ -n "$1" ] || (echo "Please supply a comma separated list of tests to be run"; exit 1)
 tests=$1
-[ ! -z "$1" ] || (echo "Please supply a username to associate with the benchmark"; exit 1)
+[ -n "$1" ] || (echo "Please supply a username to associate with the benchmark"; exit 1)
```
Comment (reltuk) [inline benchmark-tools/run_benchmarks.sh:7]:
```suggestion
[ -n "$2" ] || (echo "Please supply a username to associate with the benchmark"; exit 1)
```

**Review by reltuk — APPROVED** [review]
Looks great. Just a couple small comments.

**Pull request merged**